src/Orleans/Versions/Selector/IVersionSelector.cs

```diff
@@ -16,5 +16,15 @@ public interface IVersionSelector<TPolicy> : IVersionSelector where TPolicy : Ve
     [Serializable]
     public abstract class VersionSelectorStrategy
     {
+        public static VersionSelectorStrategy Parse(string str)
+        {
+            if (str.Equals(typeof(AllCompatibleVersions).Name))
+                return AllCompatibleVersions.Singleton;
+            if (str.Equals(typeof(LatestVersion).Name))
+                return LatestVersion.Singleton;
+            if (str.Equals(typeof(MinimumVersion).Name))
+                return MinimumVersion.Singleton;
+            return null;
```

Comment:
Does this mean you can only change the strategy at runtime if you use one of the built-in ones?